### PR TITLE
Move Dcdcboost to literate #73

### PR DESCRIPTION
**Pull request by JulienCalbert**
#73+ some updates of pathplanning

### CODE REVIEW

File: docs/src/examples/Path planning.jl [hunk 1]
```diff
@@ -5,7 +5,7 @@ using Test     #src
 #md # [![nbviewer](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/Path planning.ipynb)
 #
 # This example was borrowed from [1, IX. Examples, A] whose dynamics comes from the model given in [2, Ch. 2.4].
-# This is a **safety problem** for a **continuous system*.
+#       This is a **reachability problem** for a **continuous system**.
```
Comment (blegat) [inline docs/src/examples/Path planning.jl:8]: Why the indent ?

Comment (JulienCalbert) [reply docs/src/examples/Path planning.jl:8]: this is a mistake. For the image, I hope it will work because I had to guess its URL on Dionysos.

Comment (blegat) [reply docs/src/examples/Path planning.jl:8]: We'll see :)

File: docs/src/examples/DC-DC converter.jl [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+Xinit = AB.DomainList(Xgrid);
+union!(Xinit, Xfull)
+initlist = Int[]
+for pos in AB.enum_pos(Xinit)
+    push!(initlist, AB.get_state_by_xpos(symmodel, pos))
+end
+# Computation of the safety symbolic states:
+Xsafe = AB.DomainList(Xgrid)
+union!(Xsafe, Xfull)
+safelist = Int[]
```
Comment (blegat) [inline docs/src/examples/DC-DC converter.jl:122]: `safelist = [AB.get_state_by_xpos(symmodel, pos) for pos in AB.enum_pos(Xsafe)] `

Comment (JulienCalbert) [reply docs/src/examples/DC-DC converter.jl:122]: Indeed it's shorter :)

**Pull request merged**